src/Altinn.AccessManagement.Core/Services/PolicyInformationPoint.cs

```diff
@@ -204,76 +204,78 @@ public async Task<IEnumerable<DelegationChange>> GetOfferedDelegationsFromReposi
         public async Task<DelegationChangeList> GetAllDelegations(DelegationChangeInput request)
         {
             DelegationChangeList result = new DelegationChangeList();
-            bool validUser = DelegationHelper.TryGetUserIdFromAttributeMatch(request.Subject.SingleToList(), out int userId);
+            bool validSubjectUser = DelegationHelper.TryGetUserIdFromAttributeMatch(request.Subject.SingleToList(), out int subjectUserId);
+            bool validSubjectParty = DelegationHelper.TryGetPartyIdFromAttributeMatch(request.Subject.SingleToList(), out int subjectPartyId);
```

Comment:
Ikke helt enkelt å se på github, men dette er etter beriking av context handler?  For dette gis typisk ikke som input ved apps

Reply:
Stemmer. Endringen her i PIP API'et er bare at man får til å slå opp delegeringer gjort til CoveredByPartyId i Postgres.

PDP får til å gjøre disse kallene nå som ContextHandler også er oppdatert til å berike med PartyId til Subject